.eslintrc.js

```diff
@@ -340,5 +340,9 @@ module.exports = {
     Headers: 'readable',
     Request: 'readable',
     Response: 'readable',
+    crypto: 'readable',
+    Crypto: 'writable',
+    CryptoKey: 'writable',
+    SubtleCrypto: 'writable',
```

Comment:
There's no upside in allowing in our codebase to overwrite those.
```suggestion
    Crypto: 'readable',
    CryptoKey: 'readable',
    SubtleCrypto: 'readable',
```